packages/donor/src/components/basicV2/Button/Button.stories.tsx

```diff
@@ -0,0 +1,101 @@
+import Button, { ButtonProps } from "./Button";
+import { MuiTestWrapper } from "../../../__test__/TestUtils";
+
+export default {
+  component: Button,
+  title: "Buttonv2",
+};
+
+const baseArgs = {
+  title: "אישור והמשך",
+  onClick: () => {},
+};
+
+export const Main = (args: ButtonProps) => (
+  <MuiTestWrapper>
+    <Button {...args} />
+  </MuiTestWrapper>
```

Comment:
I changed this in master so that there's no more need for code duplication, or the test wrapper.

Please merge from master and see how Button.stories looks now, and do the same here.

(My bad for only doing this now, sorry)